lighthouse-core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -322,6 +324,31 @@ describe('TraceProcessor', () => {
       });
     });
 
+    describe('finds correct LCP', () => {
+      it('if there was a tracingStartedInPage after the frame\'s navStart', () => {
```

Comment:
not sure how to interpret this and its significance re: LCP :) Is it just saying it finds it in a valid trace?

Seems like it might be something like
```js
describe('Largest Contentful Paint', () => {
  it('finds the correct LCP in a trace', () => {
    // ...
  });
  // ...
});
```
?

Reply:
yeah, it was a copy/paste error